src/App.tsx

```diff
@@ -1,24 +1,239 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useState,
+  useEffect,
+  useMemo,
+  useCallback,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import {
+  getTodos,
+  createTodo,
+  deleteTodo,
+  toogleTodo,
+  updateTodo,
+} from './api/todos';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import {
+  TodoList,
+  Header,
+  Footer,
+  Notification,
+} from './components';
+import { warningTimer } from './utils/warningTimer';
+import { FilterTodos } from './types/FilterTodos';
+
+const USER_ID = 6336;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [creatingTodo, setCreatingTodo] = useState<Todo | null>(null);
+  const [todosInProcessed, setTodosInProcessed] = useState<Todo[]>([]);
+  const [title, setTitle] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [selectFilter, setSelectFilter] = useState<FilterTodos>(FilterTodos.ALL);
+
+  const activeTodos = todos.filter(({ completed }) => !completed);
+
```

Comment:
keep methods with methods and variables with variables, it looks more structured
```suggestion

  const activeTodos = todos.filter(({ completed }) => !completed);
  const allCompleted = todos.filter(({ completed }) => completed);
  const isAllCompleted = allCompleted.length === todos.length;
  
```